corehq/messaging/smsbackends/turn/models.py

```diff
@@ -63,12 +68,16 @@ def _send_template_message(self, client, wa_id, message_text):
             )
         except:  # TODO: Add messaging exceptions to package
             raise
+        else:
+            raise
 
-    def _send_text_message(self, client, wa_id, message_text):
+    def _send_text_message(self, client, wa_id, msg):
         try:
-            return client.messages.send_text(wa_id, message_text)
+            return client.messages.send_text(wa_id, msg.text)
         except:  # TODO: Add message exceptions to package
             raise
+        else:
```

Comment:
What is this `else` for? Wouldn't you have already exited, whether via returning or raising?

Reply:
It's just a placeholder for when the `turn-python` package has exceptions. You are correct that right now it wouldn't do anything.